geode-core/src/main/java/org/apache/geode/internal/cache/PartitionedRegionClear.java

```diff
@@ -229,9 +228,9 @@ protected void waitForPrimary(PartitionedRegion.RetryTimeKeeper retryTimer) {
         } finally {
           partitionedRegion.getDataStore().unlockBucketCreationForRegionClear();
           if (clearedBuckets.size() != 0 && partitionedRegion.getCachePerfStats() != null) {
-            partitionedRegion.getRegionCachePerfStats().incRegionClearCount();
-            partitionedRegion.getRegionCachePerfStats()
-                .incPartitionedRegionClearLocalDuration(System.nanoTime() - clearStartTime);
+            partitionedRegion.getRegionCachePerfStats().stats.incLong(CachePerfStats.clearsId, 1L);
```

Comment:
this does not look correct to me. We definitely should not be doing a CachePerfStat this way. And then the clearLocalDuration seems like it should no longer exist. So I think this method should not be doing stats. The top level clear will be measured by startClear and endClear. And the stats for each bucket will take care of showing how much time spent on buckets during a pr clear. So I don't this this method needs any stats.

Reply:
Done